docs/setup-cluster/slurm/slurm-requirements.rst

```diff
@@ -228,6 +228,94 @@ interacts with PBS, we recommend the following steps:
    define ``CUDA_VISIBLE_DEVICES`` appropriately or provide the ``pbs.slots_per_node`` setting in
    your experiment configuration to indicate how many GPU slots are intended for Determined to use.
 
+-  Configure PBS to manage GPU resources.
+
```

Comment:
When I preview the PR, it looks like there is a duplicate of this section.
<img width="797" alt="2024-01-18_12-30-17" src="https://github.com/determined-ai/determined/assets/122102764/82bbd912-d4b4-490a-9b23-83f8e70cf528">

Reply:
deleted duplicate section